server/src/Utopia/Web/Database.hs

```diff
@@ -109,7 +109,7 @@ getDatabaseConnectionString = lookupEnv "DATABASE_URL"
 createDatabasePoolFromConnection :: IO Connection -> IO DBPool
 createDatabasePoolFromConnection createConnection = do
   let keepResourceOpenFor = 10
-  createPool createConnection close 3 keepResourceOpenFor 3
+  createPool createConnection close 2 keepResourceOpenFor 1
```

Comment:
Can you add a pair of `let`s for these two numbers please since they are, or add a comment saying what they are

Reply:
Fixed in 4ce3eb773.